### PR TITLE
[DOCUMENTATION] Fix minor doc error

### PR DESCRIPTION
**Pull request by zjffdu**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ val sc = new SparkContext(conf)
 {% endhighlight %}
 
 Note that we can have more than 1 thread in local mode, and in cases like Spark Streaming, we may
-actually require one to prevent any sort of starvation issues.
+actually require more than 1 thread to prevent any sort of starvation issues.
```
Comment (rxin) [inline docs/configuration.md]: at least one?

Comment (rxin) [inline docs/configuration.md]: cc @tdas

Comment (srowen) [inline docs/configuration.md]: I think the change is correct, to my understanding. It also seems implied by the previous sentence.

Comment (andrewor14) [inline docs/configuration.md]: yeah, we need 1 for the receiver and 1 for running the jobs

**Comment by andrewor14** [comment]
Merging into master 1.6.

File: docs/configuration.md (hunk 1 above)
Comment (tdas) [inline docs/configuration.md]: yes. correct.

**Pull request closed without merging**